src/noteburst/config.py

```diff
@@ -159,6 +159,16 @@ class Config(BaseSettings):
         ),
     ] = ArqMode.production
 
+    queue_name: Annotated[
+        str,
+        Field(
+            alias="NOTEBURST_QUEUE_NAME",
+            description=(
+                "Name of the arq queue that the worker processes from."
+            ),
+        ),
+    ] = "arq:queue"
+
```

Comment:
This isn't an issue now, but _if_ we ever have noteburst run multiple container types in separate queues, then we'd need to keep the queue_name in the WorkerConfig. In which case the CronJob would be per worker stateful set. Of course we don't have any API support for multiple queues yet, but that's the original rationale here.

Reply:
Ahh good call. I lazily moved it just so I wouldn't have to fake the pod index in the Worker config if I used the worker config for the cron, but the best answer is probably to have another config just for the cron. It just needs:
* Redis connection info
* Metrics connection info
* Which queue(s) to report on

Reply:
I did this, seems cleaner.